src/main/java/seedu/cafectrl/parser/Parser.java

```diff
@@ -87,15 +90,13 @@ public static Command parseCommand(Menu menu, String userInput) {
         }
     }
 
-    // All prepareCommand Classes
+    /**
+     * All prepareCommand Classes
```

Comment:
Sorries I don't really get what does this javadoc mean? maybe you could remove it since I feel that the method is self-explanatory or elaborate more about the method if it's not trivial.

Reply:
in addition perhaps if the comment is a single line, write in the format of `/** comments */`

Reply:
This is title of the section, all the methods under this section are related to preparing command